binary_transparency/firmware/cmd/ft_monitor/main.go

```diff
@@ -54,6 +57,8 @@ func main() {
 		glog.Exitf("Failed to parse FT log URL: %q", err)
 	}
 
+	ftKeyWords := (*keyWord)
```

Comment:
no need to alias for now (we're not modifying/deriving from the value), just use the dereferenced flag later.

Reply:
Agree can do this change!